### PR TITLE
fix: bump to latest probot again

### PR DESCRIPTION
**Pull request by gr2m**
see https://github.com/probot/probot/issues/1973#issuecomment-1937111521 thanks @wolfy1339

### CODE REVIEW

File: api/index.js [hunk 1]
```diff
@@ -21,3 +21,9 @@ module.exports = (request, response) => {
 
   middleware(request, response);
 };
+
+module.exports.config = {
+  api: {
+    bodyParser: false,
+  },
+};
```
Comment (gr2m) [inline api/index.js:29]: that didn't work, I still get the singnature problems

Comment (wolfy1339) [reply api/index.js:29]: https://vercel.com/guides/how-do-i-get-the-raw-body-of-a-serverless-function

It seems that you need another function

We have similar code in Octokit,https://github.com/octokit/webhooks.js/blob/d51daf8d3f2b50cd250e367fec85d62825268f13/src/middleware/node/get-payload.ts#L30-L45
I'm not sure why it isn't working though

File: api/index.js [hunk 2]
```diff
@@ -1,16 +1,25 @@
...
-const app = require("../");
 const probot = createProbot();
-const middleware = createNodeMiddleware(app, { probot });
+const middleware = createNodeMiddleware(app, { probot, webhooksPath: "/" });
+
+export const config = {
+  api: {
+    bodyParser: false,
+  },
+};
```
Comment (gr2m) [inline api/index.js:14]: using ES Module didn't work either

**Comment by gr2m** [comment]
Setting `NODEJS_HELPERS` to `false` worked, but I'd rather disable the body parsing explicitly in code instead of relying on this environment variable, as many folks rely on `wip/app` as a reference GitHub App.

File: api/index.js (hunk 1 above)
Comment (gr2m) [reply api/index.js]: Thanks for the pointer! I'll investigate this another time: https://github.com/wip/app/issues/659

**Pull request merged**